store/tikv/kv.go

```diff
@@ -165,7 +168,7 @@ func (s *tikvStore) CheckVisibility(startTime uint64) error {
 	return nil
 }
 
-func newTikvStore(uuid string, pdClient pd.Client, spkv SafePointKV, client Client, enableGC bool) (*tikvStore, error) {
+func newTikvStore(uuid string, pdClient pd.Client, spkv SafePointKV, client Client, enableGC, enableTxnLocalLatches bool) (*tikvStore, error) {
```

Comment:
There are too many arguments to new TiKVStore.
Can we remove `enableGC` and `enableTxnLocalLatches`, and set them in a method when we want the non-default value?